tutorials/kube-mqtt/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Running a Cloud MQTT bridge in Kubernetes
+description: Bridge MQTT messages between a cluster and Cloud IoT Core with Mosquitto
+author: ptone
+tags: Kubernetes, iot, Internet of Things, cloud iot core
+date_published: 2019-07-01
+---
+
+Preston Holmes | Solution Architect | Google
+
+<!-- diagram sources: https://docs.google.com/presentation/d/1p2srfYUkqnXIKR4zGj099PIKOzz1RRi28eOcgIch1tc/edit#slide=id.p -->
+
+## Introduction
+
+This tutorial demonstrates how to deploy [Mosquitto](https://mosquitto.org/) MQTT broker in [Kubernetes](https://kubernetes.io/) using the broker's bridge feature to map a specific [MQTT](http://mqtt.org/) topic namespace to [Google Cloud IoT Core](https://cloud.google.com/iot-core/).
+
+
+## Setup
+
+### Clone the tutorial Repo
+#### TODO update to prod
```

Comment:
Note you still have TODO comment in here

Reply:
I leave that until all review is done, just a convenience to reviewers

Reply:
I'm going to do a full production and editing pass. I would prefer to have all changes like the resolution of this TODO item done before that, so that I can be approving and merging the final version at the end of my pass.